pages.js

```diff
@@ -1,20 +1,19 @@
 $(function() {
-  $(':checkbox').attr('checked', $.cookie('pjax'))
+  $(':checkbox').prop('checked', sessionStorage['pjax'])
 
-  if ( !$(':checkbox').attr('checked') )
-    $.fn.pjax = $.noop
+  if ( !$(':checkbox').prop('checked') )
+    $.fn.pjax = $.pjax.submit = $.noop
 
   $(':checkbox').change(function() {
     if ( $.pjax == $.noop ) {
```

Comment:
This condition can check for `$.support.pjax` instead which will be false if the browser doesn't have pushState support.